pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -40,6 +40,28 @@ func NewHelper() *Helper {
 func (h *Helper) InstallFlags(flags *pflag.FlagSet) {
 }
 
+// GetAllDockerAuths returns the entire docker config found, regardless of filtering
+// by imageName, and whether it was readfrom a file
+func (h *Helper) GetAllDockerAuths(authType string) (credentialprovider.DockerConfig, bool) {
```

Comment:
Why not return `error` instead of `bool`?

Reply:
I followed the existing pattern of `GetDockerAuth` which was the method that used to be employed that return only the auth for the registry in question

But I don't feel strongly about `bool` vs. `error` if you want me to change that aspect as well from the existing implementation.

So, "change" or "no change" ?

Reply:
again with the changes stemming from @mtrmac 's comments this has become a no-op